### DISCUSSION TITLE
怎么让unocss的优先级提高

### DISCUSSION THREAD

**Discussion by nigiwen** [unocss/unocss#3210] [Q&A] [answered]
### 目前的解决办法就是在组件上加上“!”来强制提升层级

![image](https://github.com/unocss/unocss/assets/67454874/10580f96-33e9-44e2-b79c-6c3bccb4c6f2)
![image](https://github.com/unocss/unocss/assets/67454874/837951d9-2a89-46aa-843d-0c607019b4a0)

**Comment by rabbit-lili** [comment]
我使用element-plus不知道为何unocss直接失效了，只有自定义的规则生效，想问问你咋配置的

**Reply by nigiwen** [reply]
可以看看[vitesse](https://github.com/antfu/vitesse)这个项目

**Comment by zyyv** [comment, marked as answer]
提升权重嘛，我觉得有多种方式都可以解决。
你也可以尝试这种
将 https://github.com/unpreset/unocss-preset-useful/blob/735cfdb199b237e7f19024915a52132dd22ba2d1/packages/core/src/core/postprocess.ts#L23-L27 放到 `postprocess` 中去
